packages/core/src/modules/credentials/CredentialsApi.ts

```diff
@@ -565,6 +569,68 @@ export class CredentialsApi<CPs extends CredentialProtocol[]> implements Credent
     }
   }
 
+  /**
+   * Send a revocation notification for a credential exchange record. Currently Revocation Notification V2 protocol is supported
+   *
+   * @param credentialRecordId The id of the credential record for which to send revocation notification
+   */
+  public async sendRevocationNotification(options: SendRevocationNotificationOptions): Promise<void> {
```

Comment:
Do you think we can make it possible to send this notification without requiring the credential record id? I think in a lot of cases you'd want to remove the credential record after you've issued so there's no PII in storage

Reply:
We are using the credential exchange record Id in order to get either the connection or services (though the corresponding request/offer messages) to send the message to. 

I think some alternatives to this approach would be to either tag connections with the `revocationId` of every credential issued to them, or create a "CredentialRevocationRecord" every time a credential is issued. The second approach is a bit more flexible because such record can contain the connectionId, the service used, etc. so it works for all cases.